app/main.py

```diff
@@ -306,31 +329,36 @@ def update_live_alerts_data_erase_buttons(n_clicks, alerts_data, alerts_frames):
     # We exclude the alerts that correspond to the event being erased
     live_alerts = live_alerts[live_alerts['event_id'] != event_id].copy()
 
-    return live_alerts.to_json(orient='records'), alerts_frames, 'hidden'
+    # Eventually, we modify the list of event_ids that must be blocked during the next updates generated by the Interval
+    blocked_event_ids['event_ids'].append(event_id)
+
+    return live_alerts.to_json(orient='records'), alerts_frames, 'hidden', blocked_event_ids
 
 
 @app.callback(
-    [Output('update_live_alerts_data_workflow', 'children'),
-     Output('update_live_alerts_frames_workflow', 'children')],
+    [Output('update_live_alerts_data_workflow', 'data'),
+     Output('update_live_alerts_frames_workflow', 'data'),
+     Output('loaded_frames', 'data'),
+     Output('main_api_fetch_interval', 'interval')],
     Input('main_api_fetch_interval', 'n_intervals'),
     [State('store_live_alerts_data', 'data'),
-     State('images_url_live_alerts', 'data')]
+     State('images_url_live_alerts', 'data'),
+     State('blocked_event_ids', 'data'),
+     State('devices_data_storage', 'data'),
+     State('loaded_frames', 'data')]
 )
-def update_live_alerts_data(n_intervals, ongoing_live_alerts, ongoing_frame_urls):
+def update_live_alerts_data(
+    n_intervals, ongoing_live_alerts, ongoing_frame_urls, blocked_event_ids, devices_data, already_loaded_frames
+):
     """
-    The following function is used to update the store containing live_alerts data from API and
-    the dictionary of images of ongoing alerts.
-
-    It returns :
-
-    - a json containing live alerts data, where live alerts either correspond to a not-yet-acknowledged event or have
-    been created less than 12 hours ago;
-
-    - a dict where keys are event id and value lists of all urls related to the same event.
-
-    These URLs come from the API calls, triggered by the intervals every 5 seconds.
+    This is the key callback of the platform. Triggered by the interval component, it queries the database via the API
+    client so as to identify live alerts, load the associated data and trigger their display on the platform. This doc-
+    string should be completed but more details can be found in the comments below.
     """
 
+    print('Number of intervals:', n_intervals)
+    print('Time:', time.time())
```

Comment:
Do we actually need that in production ?

Reply:
No, you're right, I forgot to remove it!

Reply:
Done 😊